CHANGELOG-template.rst

```diff
@@ -0,0 +1,35 @@
+0.XX.0
+------
+
+Renku ``0.XX.0`` introduces ...
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- 🐸 **Team**: changelog item with emoji, ordered by importance to users
+
+**✨ Improvements**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
+
+**🐞 Bug Fixes**
+
+- 🐸 **Team**: changelog item with emoji, alphabetized by team name
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**Improvements**
+
+- **Team**: changelog item, alphabetized by team name
+
+**Bug Fixes**
+
+- **Team**: changelog item, alphabetized by team name
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~~
+
+- component links, alphabetized by team name
```

Comment:
Should we alphabetize them in the last release too, to have it as an example?
I guess it slipped through but we can easily fix it in this PR

Reply:
You already did this... thank you... 🙇